server/src/conductor/runbooks/runtime/executors/custom-executor.ts

```
@@ -0,0 +1,406 @@
+        ? this.findPreviousOutput<unknown>(ctx, inputPath)
+        : ctx.input;
+
+      let transformedData: unknown;
+
+      switch (transformType) {
+        case 'flatten':
+          transformedData = this.flatten(inputData);
+          break;
+
+        case 'extract':
+          const extractPaths = this.getConfig<string[]>(ctx, 'extractPaths', []);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)